census/estimator/trainer/task.py

```diff
@@ -0,0 +1,162 @@
+import argparse
+import json
+import os
+
+import model
+
+from tensorflow.contrib.learn.python.learn import learn_runner
+from tensorflow.contrib.learn.python.learn.utils import (
+    saved_model_export_utils)
+from tensorflow.contrib import metrics, learn
+
+
+def generate_experiment_fn(train_file,
+                           eval_file,
+                           num_epochs=None,
+                           train_batch_size=40,
+                           eval_batch_size=40,
+                           embedding_size=8,
+                           serving_input_fn_type='JSON',
+                           hidden_units=None,
+                           **experiment_args):
+  def _experiment_fn(output_dir):
+    train_input = model.generate_input_fn(
+        train_file, num_epochs=num_epochs, batch_size=train_batch_size)
+    eval_input = model.generate_input_fn(
+        eval_file,
+        batch_size=eval_batch_size,
+        skip_header_lines=1,
+        shuffle=False
+    )
+    if args.serving_input_fn_type == 'JSON':
```

Comment:
@nikhilk @brandondutra I got this to work locally, I wasn't aware that a dictionary with a single value you can omit the key. However it's broken in the service. I'm going to leave this as JSON until we can get it working.

In the meantime. See: https://github.com/elibixby/cloudml-samples/commit/95ddb2a95234df188d4a0c7b7dbe8389845ad68d which I propose as an alternate solution once we get that bug fixed.